recipe/conda_build_config.yaml

```
@@ -5,4 +5,8 @@ mpi:
 scalar:
   - real
   - complex
-  
\ No newline at end of file
+
+device:
+  - host
+  - cuda11
+  - cuda12
```

We can avoid the skips with selectors here:

```suggestion
  - cuda11  # [linux]
  - cuda12  # [linux]
```